_schema.graphql

```diff
@@ -1522,6 +1522,16 @@ enum SubmissionSort {
   """
   CREATED_AT_DESC
 
+  """
+  sort by created_by in ascending order
+  """
+  CREATED_BY_ASC
+
+  """
+  sort by created_by in descending order
+  """
+  CREATED_BY_DESC
+
```

Comment:
do you know why created_by sorting is added in this PR? asking because this is implemented in another PR which is not merged to master yet.

Reply:
This happened due to the fact that I was pulling your branch and running the migration and not rolling it back

Reply:
that is, in my local db this field exists, therefore it is displayed in the schema